GitUI/SpellChecker/EditNetSpell.cs

```diff
@@ -510,7 +512,7 @@ private void SpellingReplacedWord(object sender, ReplaceWordEventArgs e)
         private void SpellCheckContextMenuOpening(object sender, CancelEventArgs e)
         {
             TextBox.Focus();
-            var pos = TextBox.GetCharIndexFromPosition(TextBox.PointToClient(MousePosition));
+            var pos = SpellCheckContextMenuTrigger == ToolStripMenuTrigger.Keyboard ? TextBox.SelectionStart : TextBox.GetCharIndexFromPosition(TextBox.PointToClient(MousePosition));
```

Comment:
I'd rather use `TextBox.SelectionStart` always and set the cursor on right click, too.
(I have never got why this is not automatically done by all the controls.)